searx/engines/bitchute.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""bitchute (Videos)"""
+
+from json import dumps
+from datetime import datetime
+from searx.utils import html_to_text
+
+about = {
+    "website": 'https://bitchute.com',
+    "wikidata_id": "Q45287179",
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": "JSON",
+}
+
+base_url = "https://api.bitchute.com/api/beta/search/videos"
+categories = ['videos']
+paging = True
+
+
+def request(query, params):
+
+    start_index = (params["pageno"] - 1) * 50
+    data = {"offset": start_index, "limit": 50, "query": query, "sensitivity_id": "normal", "sort": "new"}
+    params["url"] = base_url
+    params["method"] = 'POST'
+    params['headers']['content-type'] = "application/json"
+    params['data'] = dumps(data)
+
+    return params
+
+
+def response(resp):
+    search_res = resp.json()
+    results = []
+
+    for item in search_res.get('videos', []):
+
+        date = datetime.strptime(item["date_published"].split("T")[0], "%Y-%m-%d")
```

Comment:
The date may also contain a time.
```suggestion
        date = datetime.strptime(item["date_published"], "%Y-%m-%dT%H:%M:%S.%fZ")
```
Example date returned by the bitchute API I used to build this format: `2025-03-19T11:53:13.648557Z`.